integration-tests/src/test/java/com/myhome/controllers/UserControllerIntegrationTest.java

```diff
@@ -12,14 +12,17 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.context.annotation.Import;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
+import testConfig.MailTestConfig;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(SpringExtension.class)
+@Import(MailTestConfig.class)
```

Comment:
Test fails because Spring can't load props in 'MailProperties' -> all context fails when it try create 'TestRestTemplate' bean. The only one working solution i found - define 'MailProperties' bean in test config but unfortunly i can't access 'TestHelpers' from here so I copy pasted code.